JASP-Engine/JASP/R/reliabilityanalysis.R

```diff
@@ -16,24 +16,26 @@
 #
 
 ReliabilityAnalysis <- function(dataset = NULL, options, perform = "run",
-						   callback = function(...) 0,  ...) {
-
+								callback = function(...) 0,  ...) {
+	
```

Comment:
Hi @vandenman, can you please remove the trailing spaces and push the code again? (It's difficult to see the diff)